Emby.Server.Implementations/HttpServer/HttpResultFactory.cs

```diff
@@ -28,6 +28,12 @@ namespace Emby.Server.Implementations.HttpServer
     /// </summary>
     public class HttpResultFactory : IHttpResultFactory
     {
+        // Last-Modified and If-Modified-Since must follow strict date format,
+        // see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/If-Modified-Since
+        private const string HttpDateFormat = "ddd, dd MMM yyyy HH:mm:ss \"GMT\"";
+        // We specifically use en-US culture because both day of week and month names must be in it9
+        private static readonly CultureInfo _enUSculture = CultureInfo.CreateSpecificCulture("en-US");
```

Comment:
```suggestion
        private static readonly CultureInfo _enUSculture = new CultureInfo("en-US", false);
```
Any difference between these?

Reply:
Well, I took that part from MS docs on `DateTime` formatting, however this suggestion looks correct.